### DISCUSSION TITLE
I was creating a list and it got stuck.

### DISCUSSION THREAD

**Discussion by Jbordhen** [community/community#51296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/24916865/228436018-109503f5-eecf-430c-b05b-8c9e0216e174.png)
There wasn't any error in console too. After refreshing the page, the newly created list was available and my current repository was added to it.

**Comment by BasilGuo** [comment]
I also suffered from this. When I create a list from `Create list` button at top, it works well. But if I create a list from the select list, it would get stuck.

**Comment by Stalker5335** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> ![image](https://user-images.githubusercontent.com/24916865/228436018-109503f5-eecf-430c-b05b-8c9e0216e174.png)
> There wasn't any error in console too. After refreshing the page, the newly created list was available and my current repository was added to it.

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.
